File: 03. Ekspresi dan Masukan atau Keluaran/Burung Beo/BurungBeo.java
// mengimpor class Scanner dari library java.util
import java.util.Scanner;

// deklarasi class bernama BurungBeo
public class BurungBeo
{
    // titik awal eksekusi
    public static void main (String [] args) {
        {
            // membuat objek Scanner bernama scanner untuk membaca input
            Scanner scanner = new Scanner (System.in);

            String kalimat = scanner.nextLine ();

            // Menutup objek Scanner setelah selesai membaca input
            scanner.close ();
            
            // mencetak isi variabel kalimat
            System.out.print (kalimat);
        }
    }
}

File: 03. Ekspresi dan Masukan atau Keluaran/Burung Beo/Burung-Beo.cpp
// getline = fungsi untuk membaca input kalimat dari user
// sintaks = getline (std::cin, {variabel});

// memasukkan file header iostream dan string
#include <iostream>

// membuat main function (fungsi utama) program
int main ()
{
    // deklarasi dan inisialisasi variabel kalimat
    std::string kalimat;

    // meminta input, membaca input yang berbentuk kalimat, dan menyimpannya ke variabel kalimat
    getline (std::cin, kalimat);

    // mencetak isi variabel kalimat ke layar
    std::cout << kalimat;
    
    // mengakhiri program dan mengembalikan nilai 0 ke sistem operasi (nilai 0 artinya program berjalan dengan sukses)
    return 0;
}